src/elements/common/back-button/BackButton.scss

```diff
@@ -0,0 +1,8 @@
+.be-btn-back {
+    &,
+    &:hover,
```

Comment:
why do we need to specifically target hover and focus? Do we always want 5px of margin when the back button is rendered?

Reply:
The default `.btn-plain` class sets margin and padding on hover and focus for some reason, so I'm forced to here as well.